laravel/resources/js/base.js

```diff
@@ -1,7 +1,7 @@
+require('../css/main.scss');
```

Comment:
Shouldn't the main build file go in the parent directory if it's going to be loading both css and javascript?